controllers/managedocs_controller_test.go

```diff
@@ -1114,6 +1187,115 @@ var _ = Describe("ManagedOCS controller", func() {
 				}, timeout, interval).Should(Equal(ctrlutils.GetResourceRequirements("ocs-operator")))
 			})
 		})
+		When("there is a notification email address in the add-on parameter secret and smtp secret", func() {
+			It("should update alertmanager email config with smtp details", func() {
+				setupAlertmanagerConfigConditions(true, "test-key", true, "test-url", true, []string{"test@email.com"})
```

Comment:
As we do not actually care about the email address can we just specify it as a number?
Something like `number of notification email addressed`